package/gluon-luci-wifi-config/files/usr/lib/lua/luci/model/cbi/admin/wifi-config.lua

```diff
@@ -0,0 +1,90 @@
+local f, s, o, p, q
+local uci = luci.model.uci.cursor()
+
+--set the heading, button and stuff 
+f = SimpleForm("wifi", "WLAN-Config")
+f.reset = false
+f.template = "admin/expertmode"
+f.submit = "Speichern"
+
+-- text, which describes what the package does to the user
+s = f:section(SimpleSection, nil, [[
+In diesem Abschnitt hast du die Möglichkeit die SSIDs des Client- und des
+Mesh-Netzes zu deaktivieren. Bitte lass die SSID des Mesh-Netzes aktiviert, 
+damit sich auch andere Router über dich mit dem Freifunk verbinden können.
+]])
+
+local radios = {}
+
+-- look for wifi interfaces and add them to the array
+uci:foreach('wireless', 'wifi-device',
+function(s)
+  table.insert(radios, s['.name'])
+end
+)
+
+--add a client and mesh checkbox  for each interface
+for index, radio in ipairs(radios) do
+  --get the hwmode to seperate 2.4GHz and 5Ghz radios
+  local hwmode = uci:get('wireless', radio, 'hwmode')
+
+  if hwmode == '11g' or hwmode == '11ng' then --if 2.4GHz
+
+  p = f:section(SimpleSection, [[2,4GHz Wlan]], nil)
+    --box for the clientnet
+    o = p:option(Flag, 'clientbox' .. index, "Client-Netz aktivieren")
+    o.default = (uci:get_bool('wireless', 'client_' .. radio, "disabled")) and o.disabled or o.enabled
+    o.rmempty = false
+    --box for the meshnet 
+    o = p:option(Flag, 'meshbox' .. index, "Mesh-Netz aktivieren")
+    o.default = (uci:get_bool('wireless', 'mesh_' .. radio, "disabled")) and o.disabled or o.enabled
+    o.rmempty = false
+
+  elseif hwmode == '11a' or hwmode == '11na' then --if 5GHz
+
+    q = f:section(SimpleSection, [[5GHz Wlan]], nil)
+
+    --box for the clientnet
+    o = q:option(Flag, 'clientbox' .. index, "Client-Netz aktivieren")
+    o.default = (uci:get_bool('wireless', 'client_' .. radio, "disabled")) and o.disabled or o.enabled
+    o.rmempty = false
+    --box for the meshnet
+    o = q:option(Flag, 'meshbox' .. index, "Mesh-Netz aktivieren")
+    o.default = (uci:get_bool('wireless', 'mesh_' .. radio, "disabled")) and o.disabled or o.enabled
+    o.rmempty = false
+
+  end
```

Comment:
The only things that differs between the 2.4 and 5 GHz branches is the section name, please pull the identical code out of the if block.